recipe/bld.bat

```diff
@@ -1,6 +1,7 @@
 set "PREFIX_NSIS=%PREFIX%\NSIS"
 robocopy . "%PREFIX_NSIS%" /V /S /XF bld.bat /XF Docs /XF Examples
-if errorlevel 1 exit 1
+if %ERRORLEVEL% GEQ 8 exit 1
+if %ERRORLEVEL% LSS 8 exit 0
```

Comment:
Did you add the last line just to test things out? This will exit the whole script :wink:.
I'd rather use
```bat
if errorlevel 8 exit 1
call :reset_error_level
```
here with 
```bat
:reset_error_level
exit /B 0
```
appended at the end of `bld.bat`.
`if errorlevel 8` is actually already (more or less) equivalent to `if %ERRORLEVEL% GEQ 8`, i.e., it tests for `>= 8`.

Reply:
Forgot we had `exit 0` at the end already. In any event, this does seem to be working. Simplified the check. Please let me know what you think.